src/ol/color.js

```diff
@@ -42,20 +43,20 @@ export function asString(color) {
 }
 
 /**
- * Return named color as an rgba string.
+ * Return named color as an rgb(a) string.
  * @param {string} color Named color.
- * @return {string} Rgb string.
+ * @return {string} RGB(A) string.
  */
 function fromNamed(color) {
-  const el = document.createElement('div');
-  el.style.color = color;
-  if (el.style.color !== '') {
-    document.body.appendChild(el);
-    const rgb = getComputedStyle(el).color;
-    document.body.removeChild(el);
-    return rgb;
+  const name = color.toLowerCase();
+  if (!colorNames.hasOwnProperty(name)) {
+    if (name === 'transparent') {
+      return 'rgba(0,0,0,0)';
+    }
+    return '';
```

Comment:
is this a good default value? It is a string but no valid rgb(a) string. most likely i would suspect this should not cause an error, because normally if colors are allowed, an empty strings is also ok.

Also this technically differs from the previous implementation because the browsers would actually accept any color name and just parse it randomly. I think this can be neglected as it is an unspecified case - also browsers handle this differently.

Reply:
The previous behavior in this case was not really predictable.  If you `el.style.color = 'oops'`, then `getComputedStyle(el).color` depends on what other CSS rules might apply.  For example, on this page, that returns `rgb(31, 35, 40)`.  On `about:blank`, it returns `'rgb(0, 0, 0)'`.  I thought that I previously saw a case where the result was `''`, but cannot reproduce that now.

I think that the new behavior with the exported methods is probably more "correct" now.  Calling `asArray('oops')` now throws `new Error('Invalid color')` whereas previously I think the behavior would depend on CSS rules that might be on the page.